README.md

```diff
@@ -4,40 +4,119 @@
+Padertorch is designed to simplify the training of deep learning models written with [PyTorch](https://pytorch.org).
 While focusing on speech and audio processing, it is not limited to these application areas.
 
 This repository is currently under construction.
 
-The examples in contrib/examples are only working in the Paderborn NT environment
-
+[//]: <> (The examples in contrib/examples are only working in the Paderborn NT environment)
 
 # Highlights
 
-## padertorch.Trainer
-
-- Logging:
-  - The `review` of the model returns a dictionary that will be logged and visualized via 'tensorboard'. The keys define the logging type (e.g. `scalars`).
-  - As logging backend we use `TensorboardX` to generate a `tfevents` file that can be visualized from a `tensorboard`.
-- Dataset type: 
-  - lazy_dataset.Dataset, torch.utils.data.DataLoader and other iterables...
-- Validation:
-  - The `ValidationHook` runs periodically and and logs the validations results.
-- Learning rate decay with backoff:
-  - The `ValidationHook` has also parameters to do a learning rate with backoff.
-- Test run: 
-  - The trainer has a test run function to train the model for few iterations and test if
+- **Fast prototyping**: The trainer and models are all compatible with [sacred](https://github.com/IDSIA/sacred) and easily allow hyperparameter changes over the command line. Check out the [configurable](padertorch/configurable.py) module and the [examples](padertorch/contrib/examples) for how it works.
+- **Easily extensible**: Write your own network modules and models based on `padertorch.Module` and `padertorch.Model`.
+- **Seamless integration**: You provide your own data and model. We provide the trainer to wrap around your model and data. One simple command starts the training: `padertorch.Trainer.train`.
+- **Forget the training loop**: Our `padertorch.Trainer` takes care of the repetitive training loop and allows you to focus on network tuning. Included are features such as:
+  - Periodically executed validation runs
+  - Automatic checkpointing: The parameters of the model and the state of the trainer are periodically saved. The checkpoint interval and number of total checkpoints are customizable: Keep one, some or all checkpoints. We also keep track of the best checkpoint on the validation data given some metric.
+  - Learning rate scheduling
+  - Backoff: Restore the best checkpoint and change the learning rate if the loss is not decreasing.
+  - [Averaging multiple checkpoints]():
+  - [Hooks](padertorch/train/hooks.py): Extend the basic features of the trainer with your own functionality.
+- **Logging**: As logging backend, we use `TensorboardX` to generate a `tfevents` file that can be visualized from a `tensorboard`. Custom values to be logged can be defined in subclasses of `padertorch.Model`.
+- **Test run**: The trainer has a `test_run` function to train the model for few iterations and test if
     - the model is executable (burn test)
```

Comment:
The entries in this list start with lower-case "the", the other enumerations start with upper-case letters. This is a different type of enumeration as it is the continuation of a sentence. Here, I think that it sounds better if you write it like this (adding punctuation and "and"):

... test if
 - the model is executable,
 - the validation is deterministic, and
 - the model changes the parameter in the training.